lib/driver.js

```diff
@@ -484,6 +482,37 @@ export class XCUITestDriver extends BaseDriver {
     }
   }
 
+  async handleMjpegOptions() {
+    const mjpegServerPort = this.opts.mjpegServerPort || DEFAULT_MJPEG_SERVER_PORT;
+    await this.allocateMjpegServerPort(this.opts.udid,
+      mjpegServerPort, this.isRealDevice() ? true : false);
```

Comment:
`this.isRealDevice() ? true : false` is redundant. Simply use `this.isRealDevice()`